massgov/mayflower
# Patternlab/DP-13005  component display updates

## Description
Updates broken components on event and org pages in IE11

## Related Issue / Ticket

- [DP-13005](https://jira.mass.gov/browse/DP-13005)

## Steps to Test


1. IN IE11 visit /?p=pages-event and /?p=pages-organization-elected-official and check that the layouts are correct

2. Check that the /?p=pages-organization-lite-menu-variation template is no longer present
